test_data/files/server_info_test.py

```diff
@@ -6,4 +6,4 @@
 @pytest.mark.asyncio
 async def test_using_motor_client_server_info(motor_client: AsyncIOMotorClient) -> None:
     """Test retrieving the Motor client server info."""
-    await motor_client.server_info()
+    assert await motor_client.server_info() is not None
```

Comment:
Can `server_info` return `None`?

Reply:
I suppose it's better to assert if it equals to some string. Any suggestions maybe?

Reply:
Probably? The test was really just meant to be the simplest "does it work" (and by that I mean connect) test I could write. I would think the server_info would be dynamic.

Reply:
I found out that server_info will throw an exception if it can't connect via motor_client. So I rewrote the test to assert False + message if connection timeout exception was thrown. I think it adds a little more clarity to this test.